example/test_problem/Hydro/GrackleTest/yt_script/plot_projection.py

```
@@ -0,0 +1,183 @@
+         'grackle_temperature'            : (  1.0e+00,  1.0e+08 ),
+         'grackle_mu'                     : (  5.0e-01,  1.5e+00 ),
+         'grackle_cooling_rate'           : ( -1.0e-24,  1.0e-24 ),
+         'grackle_cooling_strength'       : ( -1.0e+03,  1.0e+03 ),
+         'grackle_cooling_length'         : (  1.0e-04,  1.0e+00 ),
+         'grackle_cooling_length_over_dh' : (  5.0e-02,  2.0e+01 ),
+       }
+
+
+def set_derived_fields( ds ):
+
+    def _grackle_T_over_mu( field, data ):
```

Ruff (0.15.2)

[warning] Unused function argument: `field`

(ARG001)